cmd/goal/network.go

```diff
@@ -43,7 +46,6 @@ func init() {
 
 	networkCreateCmd.Flags().StringVarP(&networkName, "network", "n", "", "Specify the name to use for the private network")
 	networkCreateCmd.Flags().StringVarP(&networkTemplateFile, "template", "t", "", "Specify the path to the template file for the network")
-	networkCreateCmd.MarkFlagRequired("template")
```

Comment:
I think we still want to support creating a network from a template.

Reply:
We will. It's just not required anymore.